dev/tests/static/framework/Magento/Sniffs/Annotation/ClassAnnotationStructureSniff.php

```diff
@@ -97,8 +97,12 @@ public function process(File $phpcsFile, $stackPtr)
     {
         $tokens = $phpcsFile->getTokens();
         $previousCommentClosePtr = $phpcsFile->findPrevious(T_DOC_COMMENT_CLOSE_TAG, $stackPtr - 1, 0);
-        $this->validateAnnotationBlockExists($phpcsFile, (int)$previousCommentClosePtr, (int)$stackPtr);
-        $commentStartPtr = (int)$phpcsFile->findPrevious(T_DOC_COMMENT_OPEN_TAG, $stackPtr - 1, 0);
+        if (!$previousCommentClosePtr) {
```

Comment:
This check should probably be `$previousCommentClosePtr === false` as I believe 0 is a valid return value from the `findPrevious` method, but should be treated differently to `false`.

Reply:
ok, but what the difference with " !" and "=== false" ?

Reply:
https://www.php.net/manual/ru/types.comparisons.php this must be the same

Reply:
@mattijv ok i agree.

Reply:
@mattijv I had this concern during review but note that it cannot be `0` here (this is a CLOSING comment pointer). Nothing to change.

Reply:
@orlangur I agree that in all practical situations the return value probably won't be 0, but personally I prefer being explicit over implicit when handling edge cases. The strict comparison with false was also the style the rest of the sniffs seemed to be favoring. However, in practice it most like won't make a difference.